.coveragerc

```diff
@@ -1010,6 +1010,7 @@ omit =
     homeassistant/components/supervisord/sensor.py
     homeassistant/components/surepetcare/__init__.py
     homeassistant/components/surepetcare/binary_sensor.py
+    homeassistant/components/surepetcare/lock.py
```

Comment:
You did add some tests already, maybe complete them instead of ignoring the new module?